### PR TITLE
update safe sysctls (v1.29)

### PR DESCRIPTION
**Pull request by hunshcn**
https://github.com/kubernetes/kubernetes/blob/v1.29.0/staging/src/k8s.io/pod-security-admission/policy/check_sysctls.go#L80
https://github.com/kubernetes/kubernetes/blob/v1.29.0/pkg/kubelet/sysctl/safe_sysctls.go#L34

### CODE REVIEW

**Comment by sftim** [comment]
/sig security

Should get a tech review: @kubernetes/sig-node-pr-reviews is this correct?

File: content/en/docs/tasks/administer-cluster/sysctl-cluster.md [hunk 1]
```diff
@@ -75,8 +75,12 @@ The following sysctls are supported in the _safe_ set:
...
 - `net.ipv4.tcp_syncookies`,
 - `net.ipv4.ping_group_range` (since Kubernetes 1.18),
-- `net.ipv4.ip_unprivileged_port_start` (since Kubernetes 1.22).
-- `net.ipv4.ip_local_reserved_ports` (since Kubernetes 1.27).
+- `net.ipv4.ip_unprivileged_port_start` (since Kubernetes 1.22),
+- `net.ipv4.ip_local_reserved_ports` (since Kubernetes 1.27, needs kernel 3.16+),
+- `net.ipv4.tcp_keepalive_time` (since Kubernetes 1.29, needs kernel 4.5+),
+- `net.ipv4.tcp_fin_timeout` (since Kubernetes 1.29, needs kernel 4.6+),
+- `net.ipv4.tcp_keepalive_intvl` (since Kubernetes 1.29, needs kernel 4.5+),
+- `net.ipv4.tcp_keepalive_probes` (since Kubernetes 1.29, needs kernel 4.5+).
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/sysctl-cluster.md:83]: nit: I'd use a semicolon (`;`) not a comma (`,`)

Also, Linux 4.14 is the oldest LTS kernel. I wonder if we can somehow account for that - maybe a **future** PR can help readers spot what is / isn't supported upstream.

Comment (hunshcn) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md:83]: Semicolon and comma formats are mixed in this repo. Do you think it should be modified here ?

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md:83]: I do, but only as a nit (less significant feedback)

Comment (hunshcn) [reply content/en/docs/tasks/administer-cluster/sysctl-cluster.md]: done

**Comment by hunshcn** [comment]
It doesn't seem that the test failed because of the change. need retest

**Comment by sftim** [comment]
This repo does not use the `/retest` Prow command (help to change that is welcome, but it's not something that SIG Docs is prioritizing).

To trigger a new build @hunshcn, you could push an empty commit.

**Comment by sftim** [comment]
Given https://github.com/kubernetes/website/pull/44355#issuecomment-1868694329

/approve

**Comment by sftim** [comment]
This tests OK locally.

**Pull request merged**